bindings/java/src/test/java/org/openucx/jucx/UcpEndpointTest.java

```diff
@@ -413,27 +413,43 @@ public void testStreamingAPI() {
             new UcpEndpointParams().setUcpAddress(worker1.getAddress()));
 
         ByteBuffer sendBuffer = ByteBuffer.allocateDirect(UcpMemoryTest.MEM_SIZE);
-        ByteBuffer recvBuffer = ByteBuffer.allocateDirect(UcpMemoryTest.MEM_SIZE);
+        sendBuffer.put(0, (byte)1);
+        ByteBuffer recvBuffer = ByteBuffer.allocateDirect(UcpMemoryTest.MEM_SIZE * 2);
+
+        UcpRequest[] sends = new UcpRequest[2];
+
+        sends[0] = clientToServer.sendStreamNonBlocking(sendBuffer, new UcxCallback() {
+            @Override
+            public void onSuccess(UcpRequest request) {
+                sendBuffer.put(0, (byte)2);
+                sends[1] = clientToServer.sendStreamNonBlocking(sendBuffer, null);
```

Comment:
BTW this could also reproduce the arbiter bug (but we can keep it for now)

Reply:
In SparkUCX arbiter, bug reproduces only on a quite big load (25+ processes, 30 cores). If i remember correctly it goes through this path when fails: https://github.com/openucx/ucx/blob/master/src/uct/ib/rc/base/rc_ep.c#L216